### PR TITLE
Test Fix LRQA-68204

### PR DESCRIPTION
**Pull request by Diogomes**

### CODE REVIEW

**Comment by Diogomes** [comment]
@natocesarregoadding this button to the macro would solve the problem, the macro "configureSubmitLabel" does not save changes made.

**Comment by natocesarrego** [comment]
Clever @Diogomes! 

Do you mind to add the `Button.clickDone();` to the macro `Form.configureSubmitLabel` and remove the occurrences of `Button.clickDone();` after all calls to `Form.configureSubmitLabel`?

Thanks

**Comment by Diogomes** [comment]
> Clever @Diogomes!
> 
> Do you mind to add the `Button.clickDone();` to the macro `Form.configureSubmitLabel` and remove the occurrences of `Button.clickDone();` after all calls to `Form.configureSubmitLabel`?
> 
> Thanks

Yes, I can.  :)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsSubmissions.testcase [hunk 1]
```diff
@@ -398,8 +398,6 @@ definition {
 
 		Form.configureSubmitLabel(submitLabel = " ");
 
-		Button.clickDone();
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsSubmissions.testcase:401]: @Diogomes, can you please do the same on `Forms.testcase`? There is one occurrence like this one. :wink:

Comment (Diogomes) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsSubmissions.testcase:401]: @natocesarrego  do I send changes made to Forms.testcase by that same LRQA?

Comment (natocesarrego) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsSubmissions.testcase:401]: Yes, @Diogomes. You can add this asked change to this same commit, no worries. :)

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**